data/upstream.yaml

```diff
@@ -395,6 +395,11 @@ python-django-haystack:
   links:
     homepage: http://haystacksearch.org/
     repo: https://pypi.python.org/pypi/django-haystack/2.5.0
+python-django-openid-auth:
+  links:
+    homepage: https://launchpad.net/django-openid-auth
+    repo: https://pypi.python.org/pypi/django-openid-auth
+    bug: https://caniusepython3.com/project/django-openid-auth
```

Comment:
Sorry for the delay (blame summer!)

Where does the caniusepython3.com link come from? As far as I can see it's not linked from the project.